### PR TITLE
Add a basic travis configuration.

### PR DESCRIPTION
**Pull request by tomprince**

### CODE REVIEW

File: src/crtsh.rs [hunk 1]
```diff
@@ -1,11 +1,12 @@
+
+
```
Comment (alex) [inline src/crtsh.rs]: Is the recommended format really two blank lines at the top of a file?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+         cargo install-update -i clippy
+      else
+         cargo install --force clippy
+      fi
+    fi
+  - cargo install-update -i rustfmt
+script:
+  - |
+    RUSTFLAGS=-Dwarnings cargo build &&
+    cargo test
```
Comment (alex) [inline .travis.yml]: This can all be on one line, and it'll improve readability.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  - |
+    cargo fmt -- --write-mode diff
```
Comment (alex) [inline .travis.yml]: The `|` isn't needed here.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  - |
+    [[ ! $TRAVIS_RUST_VERSION =~ nightly-* ]] || RUSTFLAGS=-Dwarnings cargo clippy
```
Comment (alex) [inline .travis.yml]: Same

File: src/crtsh.rs (hunk 1 above)
Comment (tomprince) [inline src/crtsh.rs]: Nah. That is an artifact of reordering imports. I've fixed them.

File: .travis.yml (hunk 4 above)
Comment (tomprince) [inline .travis.yml]: A leading `[` is special to yaml.

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+      if type -p cargo-clippy && cargo-clippy -V; then
+         cargo install-update -i clippy
+      else
+         cargo install --force clippy
+      fi
+    fi
+  - cargo install-update -i rustfmt
+script:
+  - RUSTFLAGS=-Dwarnings cargo build && cargo test
+    cargo fmt -- --write-mode diff
```
Comment (alex) [inline .travis.yml]: This needs it's own `-` , I assume.

**Comment by alex** [comment]
Thanks!

**Pull request merged**